source/examples/qtexample/WindowQt.cpp

```diff
@@ -170,6 +170,7 @@ void WindowQt::resizeGL(QResizeEvent *)
 
 void WindowQt::paintGL()
 {
```

Comment:
this function being empty threw me off initially, hence I was calling paint(). Better to either get rid of this function or route thru updateGL() which works correctly.